notebooks/astrocut/roman-cutouts-with-astrocut/roman-cutouts-with-astrocut.ipynb

```diff
@@ -0,0 +1,207 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "77bdd3ee",
+   "metadata": {},
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d107a95c",
+   "metadata": {},
+   "source": [
+    "The Roman Space Telescope will introduce a new file type that will be used to store astronomical data in a seemless fashion. This file type is called ASDF which is short for [TODO].\n",
+    "\n",
+    "The `astrocut` service allows users to generate cutouts out of large images from survey sattelites such as the Transiting Exoplanet Survey Satellite (TESS). Future plans for `astrocut` involves expanding its functionality so that it supports generating cutouts from different missions, including the Roman Space Telescope mission, which will involve supporting the ASDF file type.\n",
```

Comment:
```suggestion
    "The `astrocut` service allows users to generate cutouts out of large images from survey satellites such as the Transiting Exoplanet Survey Satellite (TESS). Future plans for `astrocut` involves expanding its functionality so that it supports generating cutouts from different missions, including the Roman Space Telescope mission, which will involve supporting the ASDF file type.\n",
```